pkg/controller/controller_test.go

```diff
@@ -664,13 +665,32 @@ func (s *ControllerSuite) TestDeferPhase(c *C) {
 	err = s.waitOnActionSetState(c, as, crv1alpha1.StateRunning)
 	c.Assert(err, IsNil)
 
+	onPhases := sets.NewString()
```

Comment:
I don't think we need set, here. I will update this to just have slice of strings.